apps/minifront/src/components/swap/duration-slider.tsx

```diff
@@ -27,12 +27,16 @@ export const DurationSlider = () => {
         onValueChange={handleChange}
       />
 
-      <div className='flex flex-col items-center'>
-        ~ {duration}{' '}
-        <span className='text-xs text-muted-foreground'>
-          distributed across {GDA_RECIPES[duration].numberOfSubAuctions.toString()} sub-auctions
-        </span>
-      </div>
+      {duration === 'instant' && <>instant</>}
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/5434e13e-1067-4fdd-9913-9eddafbb2324)

(Design to be improved soon)